homeassistant/components/deconz/__init__.py

```diff
@@ -75,8 +77,14 @@
             "Config entry failed since one deCONZ instance already exists")
         return False
 
+    @callback
+    def async_add_device_callback(device_type, device):
+        """Called when a new device has been created in deCONZ."""
+        async_dispatcher_send(hass, DOMAIN + '_new_' + device_type, device)
```

Comment:
Don't use `+` for string concatenation, always use the string format function `"{}_new_{}".format(DOMAIN, device_type)`. Also, why not hardcode DOMAIN 🤔  ?

Reply:
You ask the hard questions ;), I think I have some kind of disease in trying to parameterise everything